resolwe_bio/processes/junctions/regtools.yml

```diff
@@ -10,9 +10,12 @@
     executor:
       docker:
         image: resolwebio/rnaseq:3.7.0
-  data_name: "Novel Junctions ({{ star|sample_name|default('') }}{{ extracted|sample_name|default('') }})"
-  version: 0.0.1
-  type: data:bed:junctions
+  data_name: >
+    "Novel Junctions ({{ alignment_star|sample_name|default('?') }}
+    {{ alignment|sample_name|default('?') }}
+    {{ input_bed_junctions.bed.file|default('?') }})"
+  version: 0.0.2
+  type: data:junctions:regtools
```

Comment:
What is the reason for changing the type? Just curious.

Reply:
I have changed it, that it is structured in the same way as `types` in other processors.
For example: ``data:expression:featurecounts`` or ``data:chipseq:macs14`` or ``ata:differentialexpression:deseq2``, in all these cases `type` consists of 'data:category:tool'.
So I have changed it in the same way: `data:junctions:regtools`